src/components/PostList/PostList.jsx

```diff
@@ -1 +1,9 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} />
```

Comment:
You forgot to add a `key` to the `PostInfo` tag. According to the checklist, a `key` should be added to the root tag inside an iteration. The `key` helps React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity. Here is how you can do it: `<PostInfo post={post} key={post.id} />`